elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -454,6 +456,52 @@ export abstract class BaseTransaction {
 		this._id = getId(this.getBytes());
 	}
 
+	/* tslint:disable:next-line: no-any no-null-keyword */
+	public dbRead(raw: any): TransactionJSON | null {
+		if (!raw.t_id) {
+			return null;
+		}
+
+		const transactionJSON: TransactionJSON & {
+			readonly requesterPublicKey: string;
+			readonly [key: string]: string | number | object | null;
+		} = {
+			id: raw.t_id,
+			height: raw.b_height,
+			blockId: raw.b_id || raw.t_blockId,
+			type: parseInt(raw.t_type, 10),
+			timestamp: parseInt(raw.t_timestamp, 10),
+			senderPublicKey: raw.t_senderPublicKey,
+			requesterPublicKey: raw.t_requesterPublicKey,
```

Comment:
I think this key doesn't exist, so we can remove?
please check on the storage

Reply:
I thought so too but it still exists in `storage/entities/transactions` line 135. Also the original `dbRead` function included it.